server/src/test/scala/sbt/server/ProtocolTest.scala

```diff
@@ -621,13 +624,13 @@ class ProtocolTest {
           case ObjectToJson =>
             val json = addWhatWeWerePickling(t)(SerializedValue(t)(format.pickler))
             // IO.write(path, json.toString, IO.utf8)
-            if (path.exists) sys.error(s"$path exists already")
+            if (path.exists) () // Just ignore things we write twice, reading into different types should work.
```

Comment:
I think the issue here is that I used the same seq_foo.json filename for all of list, vector, seq tests, so when josh went to regen the protocol json files he hit this check. Reusing the filename is really only valid to do if we emit the same JSON for all of them, I think, because if we pickled them differently we'd need to test them separately. So we could check that when we generate the json files.

However, it would be simpler to just make list_foo.json and vector_foo.json separately though instead of getting fancy, I just made a mess by not doing that.